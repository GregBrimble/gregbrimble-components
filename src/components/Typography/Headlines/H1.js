import React from "react"

import styled from "styled-components"
import { Headline } from "./shared"

const H1 = props => (
  <Headline
    as={styled.h1`
      font-size: 6rem;
      line-height: 6rem;
    `}
    {...props}
  />
)

export default H1
