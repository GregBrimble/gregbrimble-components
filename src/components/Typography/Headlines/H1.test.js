import React from "react"
import H1 from "./H1"
import renderer from "react-test-renderer"

describe("H1", () => {
  it("renders properly", () => {
    const tree = renderer.create(<H1>Headline 1</H1>).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
