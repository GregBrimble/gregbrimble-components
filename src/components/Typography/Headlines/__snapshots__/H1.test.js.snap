// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H1 renders properly 1`] = `
<h1
  className="sc-bdVaJa hEOvXI sc-htpNat cXPedG"
>
  Headline 1
</h1>
`;
